bin/main

```diff
@@ -1,3 +1,61 @@
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+puts '1 | 2 | 3'
+puts '---------'
+puts '4 | 5 | 6'
+puts '---------'
+puts '7 | 8 | 9'
+
+puts "\nPlayer 1 (X), insert your name:"
+player_1 = gets.chomp # rubocop:todo Naming/VariableNumber
+
+puts "\nPlayer 2 (O), insert your name:"
+player_2 = gets.chomp # rubocop:todo Naming/VariableNumber
```

Comment:
- Adding these rubocop rules could have been avoided if you name the player variable as 'player_one'. :pray:  remove these rules